_includes/setup/android-setup-linux.md

```diff
@@ -0,0 +1,88 @@
+## Android setup
+
+_This step is optional, and can be performed at any time before
+using Flutter to develop apps for Android._
+
+### Install Android Studio
+
+To develop Flutter apps for Android, you need to install Android Studio.
+
+1. Install [Android Studio](https://developer.android.com/studio/index.html).
+
+   Flutter requires the latest Android SDK and Android SDK Platform-Tools packages.
+   You can verify these are up-to-date in **Android Studio>Configure>SDK Manager**,
+   as described in [Update the IDE and SDK Tools](https://developer.android.com/studio/intro/update.html).
+
+1. Start Android Studio.
+
+### Set up your Android device
+
+To prepare to run and test your Flutter app on Android, you'll need an Android device running
+Android 4.1 (API level 16) or higher.
+
+1. Enable developer mode on your device by visiting **`Settings>About phone`** and
+tapping the **Build number** line seven times.
+2. In **Settings>Developer options**, enable **USB debugging**.
+3. Using a USB cable, plug your phone into your computer. If prompted on your
+device, authorize your computer to access your device.
+4. In the terminal, run the `flutter devices` command to verify that Flutter recognizes your
+connected Android device.
+5. Start your app by running `flutter run`.
+
+By default, Flutter uses the version of the Android SDK where your `adb` tool is based. If
+you want Flutter to use a different installation of the Android SDK, you must set the
+`ANDROID_HOME` environment variable to that installation directory.
+
+### Set up the Android emulator
+
+To prepare to run and test your Flutter app on the Android emulator, follow these steps:
+1. Enable Kernel-based Virtual Machine (KVM) support on your machine.
+1. Define a virtual device in Android Virtual Device Manager.
+
+#### Enable KVM
```

Comment:
I think this is getting too detailed, and IMHO getting outside what we should document vs rely on documentation of our partners. I would just have a small line of text saying hardware accelleration is recommended and then link to Android docs: https://software.intel.com/en-us/android/articles/speeding-up-the-android-emulator-on-intel-architecture

You may also note in these docs:
1. There are steps for all three OS'es we support
2. The steps are different for Intel and AMD, so don't think the current docs we have will work on AMD (that would not have *Intel* VT-d